app/src/main/java/org/mozilla/focus/utils/Settings.kt

```diff
@@ -118,12 +126,19 @@ class Settings(
                 .commit()
         }
 
-    fun shouldBlockImages(): Boolean =
-        // Not shipping in v1 (#188)
-            /* preferences.getBoolean(
-                    resources.getString(R.string.pref_key_performance_block_images),
-                    false); */
-        false
```

Comment:
Nit: Let's refactor apply refactors in separate ticket as it makes the diff harder to read